config/local-development/tilt/add-vault-environment-variables.yaml

```diff
@@ -1,5 +1,5 @@
 - op: add
-  path: /spec/template/spec/containers/1/env
```

Comment:
does this still not work? I cannot accept this. After the new rebase this should have been fixed for everyone.

Reply:
only when using path: /spec/template/spec/containers/0/env
does the tilt up command work for me.
What version of kustomize are you using?
Here mine...
[tbox@fedora vault-config-operator]$ kustomize version
{Version:kustomize/v4.4.1 GitCommit:b2d65ddc98e09187a8e38adc27c30bab078c1dbf BuildDate:2021-11-11T23:36:27Z GoOs:linux GoArch:amd64}

Reply:
Seems like we need to fix this in a separate PR.